packages-ui/roosterjs-react/lib/ribbon/component/getButtons.ts

```diff
@@ -34,6 +34,7 @@ import { underline } from './buttons/underline';
 import { undo } from './buttons/undo';
 
 const KnownRibbonButtons: { [key in KnownRibbonButtonKey]: RibbonButton<AllButtonStringKeys> } = {
+    [KnownRibbonButtonKey.InsertLink]: insertLink,
```

Comment:
any reason to move this button to the first?

Reply:
For testing, I will revert it back when this PR is ready for review.